Jenkinsfile

```diff
@@ -96,6 +103,10 @@ node('rhel8'){
 		def vsix = findFiles(glob: '**.vsix')
 		// VS Code Marketplace
 		withCredentials([[$class: 'StringBinding', credentialsId: 'vscode_java_marketplace', variable: 'TOKEN']]) {
+			def platformVsixes = findFiles(glob: '**.vsix', excludes: vsix[0].path)
+			for(platformVsix in platformVsixes){
+				sh 'vsce publish -p ${TOKEN}' + " --packagePath ${platformVsix.path}"
+			}
 			sh 'vsce publish -p ${TOKEN} --packagePath' + " ${vsix[0].path}"
```

Comment:
Just one last thing to change here. We need to explicitly provide `--target` on the generic version for all the platforms we do not explicitly support. At least, that's we did on the vscode-java side.

eg. `vsce publish -p ${TOKEN} --target win32-ia32 win32-arm64 linux-armhf alpine-x64 alpine-arm64`